src/syntax_util.ml

```diff
@@ -72,6 +72,7 @@ let reason_to_ml_swapping_alist = [
   "\\===", "===";
   "!=", "<>";
   "!==", "!=";
+  "\\/><", "/><";
```

Comment:
I tested the operators and the other two already failed without the PR, will re-check.